packages/react-compass/src/textfield/textfield.tsx

```diff
@@ -149,7 +149,17 @@ const TextField = React.forwardRef<HTMLDivElement, TextFieldProps>(
     }, [password, isPassWordVisible])
 
     return (
-      <StyledTextFieldWrapper css={css} {...htmlProps} ref={textfieldRef}>
+      <StyledTextFieldWrapper
+        css={css}
+        {...htmlProps}
+        ref={textfieldRef}
+        onKeyDown={(e) => {
+          e.preventDefault()
+        }}
+        onKeyUp={(e) => {
+          e.preventDefault()
+        }}
+      >
```

Comment:
This isn't how to solve this problem. The root cause for this bug is duplicate props passed to both wrapper and input elements thus onKeyDown can be triggered twice

Reply:
I've done it for you